airflow/providers/amazon/aws/operators/emr_create_job_flow.py

```diff
@@ -40,7 +42,7 @@ class EmrCreateJobFlowOperator(BaseOperator):
     :type region_name: Optional[str]
     """
 
-    template_fields = ['job_flow_overrides']
+    template_fields = ['job_flow_overrides' , 'aws_conn_id']
```

Comment:
Looks like this extra whitespace is triggering ther CI test failures.  Runnig ` pre-commit run --all-files` should auto-format most/all of that out for you using `black` and `flake8`.

Reply:
fixed and commited back.